content/guides/models/track/log/distributed-training.md

```diff
@@ -24,15 +24,18 @@ Explore the code behind these examples in the W&B GitHub examples repository [he
 
 This section describes how to track values and metrics available to your rank 0 process. Within the rank 0 process, initialize a W&B run with [`wandb.init`]({{< relref "/ref//python/init.md" >}}) and log experiments ([`wandb.log`]({{< relref "/ref//python/log.md" >}})) to that run.
 
-W&B does not track metrics from other nodes or processes, such as loss values or inputs from training batches. W&B does track system metrics, such as usage and memory, for all GPUs since that information is available to all processes.
+W&B tracks system metrics, such as usage and memory, for all GPUs since that information is available to all processes. W&B does not track metrics from other nodes or processes, such as loss values or inputs, from training batches. 
 
 {{% alert %}}
-**Use this approach to only track metrics available from a single process**. Typical examples include GPU/CPU utilization, behavior on a shared validation set, gradients and parameters, and loss values on representative data examples.
+**Use this approach to track only metrics that are available from a single process**. Typical metrics include GPU/CPU utilization, behavior on a shared validation set, gradients and parameters, and loss values on representative data examples.
 {{% /alert %}}
```

Comment:
The para before the alert is difficult to parse, and I feel that the first sentence is a run on. I think what you are trying to say is that  no action is needed to get system metrics for the process its node, but that processes on other nodes are opaque.

The details inside the alert partially duplicate the details about the types of metrics. I am not sure that the detail about typical metrics belongs in an alert.